pkgs/tools/inputmethods/droidmote/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  srcs = {
+    x86_64-linux = fetchurl {
+      url = "https://videomap.it/script/dms-ubuntu-x64";
```

Comment:
I really don't like those unversioned URLs. If this package is updated this derivation will stop working.

Can we have a versioned one instead 🤔 ?

Reply:
If upstream doesn't offer a versioned URL please use archive.org.

Reply:
hm, this is going to be difficult since I got the links inside a shell script downloaded through the website. Do u think the internet archive has access to those links too?

Reply:
![image](https://user-images.githubusercontent.com/29521461/161394971-037c7ab5-d8c0-4711-b9d0-fe5b1e5b030f.png)

Reply:
A solution the guys from AUR used were to store those binaries in a cloud storage provider, but I dont think this should be our solution.

Reply:
> ![image](https://user-images.githubusercontent.com/29521461/161394971-037c7ab5-d8c0-4711-b9d0-fe5b1e5b030f.png)

This is normal. Click on the `Save this url in the Wayback Machine` and it should generate archive a new snapshot.

As long as the site doesn't have e.g. a `robots.txt` disallowing Wayback Machine crawler, this should work. If this still doesn't work, let's find another solution.

> A solution the guys from AUR used were to store those binaries in a cloud storage provider, but I dont think this should be our solution.

Instead of `url`, you can also use `urls` that accept a list of URLs. As long as at least one of those URLs matches the `sha256`, the build will work.

So we can keep the current URL, add as a fallback the binaries from AUR and https://archive.org link.